src/igvfd/schemas/changelogs/sequence_file.md

```diff
@@ -1,7 +1,7 @@
 ## Changelog for *`sequence_file.json`*
 
 ### Minor changes since schema version 14
-
+* Extend `sequencing_kit` enum list to include `AVITI 2x75 Sequencing Kit Cloudbreak High Output`.
```

Comment:
Can you restore the blank line under the header? (For example how the Platform Term changelog looks)

Reply:
Updated